sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/ParquetFileDataObject.scala

```diff
@@ -93,12 +93,6 @@ case class ParquetFileDataObject( override val id: DataObjectId,
 
   override val options: Map[String, String] = parquetOptions.getOrElse(Map())
 
-  // when writing parquet files, schema column names are forced to lower,
-  // because they can also be accessed by Hive which is case insensitive.
-  // See: https://medium.com/@an_chee/why-using-mixed-case-field-names-in-hive-spark-sql-is-a-bad-idea-95da8b6ec1e0
```

Comment:
This issue was fixed in Spark 2.4, see https://spark.apache.org/docs/latest/sql-migration-guide.html#:~:text=when%20spark.sql.caseSensitive%20is%20set%20to%20false%2C%20Spark%20does%20case%20insensitive%20column%20name%20resolution%20between%20Hive%20metastore%20schema%20and%20Parquet%20schema